pytest.ini

```
@@ -0,0 +1,17 @@
+[pytest]
+addopts =
+    # show 10 slowest invocations:
+    --durations=10
+    # a bit of verbosity doesn't hurt:
+    -v
+    # report all the things == -rxXs:
+    -ra
+    # show values of the local vars in errors:
+    --showlocals
+    # coverage reports
+    --cov=aiohttp_sse/ --cov=tests/ --cov-report term
+asyncio_mode = auto
+filterwarnings =
+    error
```

Since we should support aiohttp<3.9:

```suggestion
filterwarnings =
    error
    ignore::aiohttp.web_exceptions.NotAppKeyWarning
```